content/ngf/installation/installing-ngf/deploy-data-plane.md

```diff
@@ -0,0 +1,244 @@
+metadata:
+  name: cafe
+spec:
+  gatewayClassName: nginx
+  listeners:
+  - name: http
+    port: 80
+    protocol: HTTP
+EOF
+```
+
+To check that the Gateway has deployed correctly, use `kubectl describe` to check its status:
+
+```shell
+kubectl describe gateway
+```
+
+You should see these conditions:
+
+```text
+Conditions:
+      Last Transition Time:  2025-05-05T23:49:33Z
+      Message:               Listener is accepted
+      Observed Generation:   1
+      Reason:                Accepted
+      Status:                True
+      Type:                  Accepted
+      Last Transition Time:  2025-05-05T23:49:33Z
+      Message:               Listener is programmed
+      Observed Generation:   1
+      Reason:                Programmed
+      Status:                True
+      Type:                  Programmed
+      Last Transition Time:  2025-05-05T23:49:33Z
+      Message:               All references are resolved
+      Observed Generation:   1
+      Reason:                ResolvedRefs
+      Status:                True
+      Type:                  ResolvedRefs
+      Last Transition Time:  2025-05-05T23:49:33Z
+      Message:               No conflicts
+      Observed Generation:   1
+      Reason:                NoConflicts
+      Status:                False
+      Type:                  Conflicted
+```
+
+Using `kubectl get` we can see the NGINX Deployment:
+
+```text
+~ ❯ kubectl get deployments                                                                                                                                        ⎈ kind-kind
+NAME         READY   UP-TO-DATE   AVAILABLE   AGE
+cafe-nginx   1/1     1            1           3m18s
+```
+
+We can also see the Service fronting it:
+
+```text
+~ ❯ kubectl get services                                                                                                                                           ⎈ kind-kind
+NAME         TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)        AGE
+cafe-nginx   NodePort    10.96.125.117   <none>        80:30180/TCP   5m2s
+```
+
+The type of Service can be modified, which will be explained below.
+
+---
+
+## How to modify provisioned NGINX instances
+
+Both the NGINX Deployment and Service Kubernetes objects provisioned by NGINX Gateway Fabric upon creation of a Gateway
+can be modified through the NginxProxy custom resource. 
+
+{{< note >}} Updating most Kubernetes related fields in NginxProxy will trigger a restart of the related resource to update. {{< /note >}}
+
+An NginxProxy resource is created by default after deploying NGINX Gateway Fabric. Use `kubectl get` and `kubectl describe` to 
+get some more information on the resource:
+
+```text
+~ ❯ kubectl get nginxproxies -A                                                                                                                                    ⎈ kind-kind
+NAMESPACE       NAME                      AGE
+nginx-gateway   my-release-proxy-config   19h
```

Comment:
Since we install NGF with the `ngf` release name in our installation guide, can you use that release name for consistency? There may be other places in the docs that need to be fixed too.